.azure/pipelines/fast-pr-validation.yml

```diff
@@ -11,14 +11,15 @@ resources:
     name: aspnet/BuildTools
```

Comment:
Thanks for copying this over. You can remove the whole `resources:` section because we don't reference remote templates anymore